# HG changeset patch
# User Jason Kim <jasonwkim@google.com>
# Date 1301356914 25200
# Branch pnacl-sfi
# Node ID 985777e2e62f6daca9aec8dab851eb61715f045f
# Parent 3798256be70e3cbe281acf7e82c3ecd6ee50f864
Fix minor bug to enable relocation Fixups in DWARF data.

 From llvm-pnacl-0016-236-985777e2e62f6daca9aec8dab851eb61715f045f.patch

 From .hg/patches/llvm-pnacl-0016-236-985777e2e62f6daca9aec8dab851eb61715f045f.patch.stripped

diff -r 3798256be70e lib/Target/ARM/ARMAsmBackend.cpp
--- a/lib/Target/ARM/ARMAsmBackend.cpp	Fri Mar 18 08:34:35 2011 -0700
+++ b/lib/Target/ARM/ARMAsmBackend.cpp	Fri Jul 29 13:13:09 2011 -0700
@@ -410,7 +410,7 @@
   if (!Value) return;           // Doesn't change encoding.
 
   unsigned Offset = Fixup.getOffset();
-  assert(Offset % NumBytes == 0 && "Offset mod NumBytes is nonzero!");
+  // assert(Offset % NumBytes == 0 && "Offset mod NumBytes is nonzero!");
 
   // For each byte of the fragment that the fixup touches, mask in the bits from
   // the fixup value. The Value has been "split up" into the appropriate
